source/_static/css/custom.css

```
@@ -610,4 +610,10 @@ div.endpoint-list div.panel-body .container {
   padding: 0;
 }
 
+.response-col_description__inner,
+.response-control-media-type,
+.response-col_links
+{
+  display: none;
+}
```

I have been putting any css rules that target the swagger-ui specifically into `css/swagger-ui/index.css` and targeting them with `.swagger-ui` ie 

```suggestion
.swagger-ui .response-col_description__inner,
.swagger-ui .response-control-media-type,
.swagger-ui .response-col_links
{
  display: none;
}
```

That way we don't accidentally target any other html elements outside of the embedded swagger ui